From 635343d5e94856739bd12458b20f53d2f8ddc8b7 Mon Sep 17 00:00:00 2001
From: olegendo <olegendo@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Sat, 3 May 2014 08:17:43 +0000
Subject: [PATCH] 	Back port from mainline
 	PR target/61026
 	* config/sh/sh.c: Include stdlib headers before everything else.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_9-branch@210029 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/config/sh/sh.c |    8 ++++----
 2 files changed, 10 insertions(+), 4 deletions(-)

diff --git a/gcc/config/sh/sh.c b/gcc/config/sh/sh.c
index 6d909c7..12724a2 100644
--- a/gcc/config/sh/sh.c
+++ b/gcc/config/sh/sh.c
@@ -19,6 +19,10 @@ You should have received a copy of the GNU General Public License
 along with GCC; see the file COPYING3.  If not see
 <http://www.gnu.org/licenses/>.  */
 
+#include <sstream>
+#include <vector>
+#include <algorithm>
+
 #include "config.h"
 #include "system.h"
 #include "coretypes.h"
@@ -70,10 +74,6 @@ along with GCC; see the file COPYING3.  If not see
 #include "pass_manager.h"
 #include "context.h"
 
-#include <sstream>
-#include <vector>
-#include <algorithm>
-
 int code_for_indirect_jump_scratch = CODE_FOR_indirect_jump_scratch;
 
 /* These are some macros to abstract register modes.  */
-- 
1.7.1

